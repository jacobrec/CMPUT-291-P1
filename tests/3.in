1
testy
password
3
10000
3
0
3
16
3
17
7
2

