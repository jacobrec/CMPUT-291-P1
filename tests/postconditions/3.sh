echo "SELECT expiry FROM registrations WHERE regno=16" | sqlite3 src/sql/test.db
echo "SELECT expiry FROM registrations WHERE regno=17" | sqlite3 src/sql/test.db
