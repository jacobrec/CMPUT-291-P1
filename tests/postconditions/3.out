expiry
----------
2020-11-04
expiry
----------
2021-08-08
